Client/Application/Delegates/AppDelegate.swift

```diff
@@ -678,7 +678,10 @@ extension AppDelegate {
                 return false
             }
 
-            browserViewController.switchToTabForURLOrOpen(url, isPrivileged: false)
+            browserViewController.switchToTabForURLOrOpen(
+                url,
+                isPrivate: Preferences.Privacy.privateBrowsingOnly.value ? true :  false,
```

Comment:
can't just write it like
`isPrivate: Preferences.Privacy.privateBrowsingOnly.value`?

Reply:
Fix at https://github.com/brave/brave-ios/pull/3973/commits/cf0498e296d6a7db8bdb6d45b0f0b2ee044349b5